fhir/README.adoc

```diff
@@ -0,0 +1,72 @@
+= FHIR: A Camel Quarkus example
+:cq-example-description: An example that shows how to use Camel FHIR with Quarkus.
+
+{cq-description} It also shows how you can build custom health-checks and have them automatic discovered by Camel and
+used as parts of its health-check system.
+
+TIP: Check the https://camel.apache.org/camel-quarkus/latest/first-steps.html[Camel Quarkus User guide] for prerequisites
+and other general information.
+
+The example has two routes. The timer route performs a given task at regular interval. For the sake of this example we'll say
+that this task unexpectedly freezes the service from time to time. The details can be consulted at http://localhost:8080/q/health.
+The custom health check is expected to report UNKNOWN on first consultation, then UP during 10 seconds, and finally DOWN so as to
+simulate that the service is frozen.
+
+The 2nd route is on purpose made to fail on startup by configuring netty to an unknown host.
+Camel supervising route controller will attempt to restart the route up till 10 times before exhausting.
+The routes health check will therefore report this route as DOWN until its exhausted where the states are changed to UNKNOWN.
+
+The details can be seen at runtime via the following url from a web browser: http://localhost:8080/q/health.
```

Comment:
You can remove this and copy most of the introduction content from the Spring Boot example README (avoiding any SB specific references).